### DISCUSSION TITLE
Howto Search Subgroups including titles

### DISCUSSION THREAD

**Discussion by darnea94** [keepassxreboot/keepassxc#7091] [Q&A] [answered]
Hey,

I've recently switched from KeePass to KeePass XC on my new Mac. The Entrys are in the following order :

Client ABC
-> User
--> EntryABC
Customer XYZ
-> User
--> EntryABC

If I wanted to get the result of EntryABC of Customer XYZ in KeePass I had to search for example"Customer Entry" "Customer ABC" "XYZ entry"

How can I accomplish that in KeePass XC. With the g: operator I can select the group, but not the Subgroup so if I search for "g:Customer XYZ Entry" I will find nothing.

Thank you in Advance !

**Comment by droidmonkey** [comment, marked as answer]
Use quotes around the search term or just use the xyz part. `g:"customer xyz" entry`

**Reply by darnea94** [reply]
That's the problem. The Subfolder "User" will not be listed in the search. When I'm using your search term no results will be found. If I'm searching for "entry" all Entrys of all Customers/Groups will be shown.

**Reply by droidmonkey** [reply]
Then add /User after the parent group name

**Reply by darnea94** [reply]
thank you. I can search the database by g:customername/* entry name.
 
you are the best
